### ISSUE TITLE
Added new EdgeKind: ref/file [phab:D1650]

### ISSUE THREAD

**Issue by kythe** [google/kythe#2074]
_Created by **craigbarber** at 2017-05-17 22:49:28_: 

Added new EdgeKind: ref/file to the schema. This is necessary for indexers, such as the bazel indexer, which have anchors referencing files directly.

**Comment by kythe** [comment]
_**zarko** commented inline (context lost during migration) at 2017-05-17 22:56:21_:

Add:
```
This is distinct from <<refincludes,[ref/includes]>>, which indicates that the anchor causes the contents of `F` to be inserted into the surrounding file. *ref/file* should be used when the anchor refers explicitly to the identity of `F`, as in a hardcoded path in a test or build script.
```

**Comment by kythe** [comment]
_**craigbarber** wrote at 2017-05-18 17:25:01_:

  - Incorporated review feedback

**Comment by kythe** [comment]
_**zarko** accepted this revision at 2017-05-18 19:51:58_

**Comment by kythe** [comment]
_**craigbarber** closed via commit 4b107f7bfd8ebb8334ae6a74a5fcf382f182ef7e at 2017-05-18 20:06:08_
